tests/multi-index-search.tests.ts

```diff
@@ -0,0 +1,270 @@
+    const response = await gamesIndex.addDocuments(games)
+
+    await meilisearchClient.waitForTask(response.taskUid)
+  })
+
+  test('searching on two indexes', async () => {
+    const customClient = instantMeiliSearch(
+      'http://localhost:7700',
+      'masterKey'
+    )
+
+    const response = await customClient.search<Movies>([
+      {
+        indexName: 'movies',
+        params: {
+          query: 'c',
+        },
+      },
+      {
+        indexName: 'games',
+        params: {
+          query: 'c',
+        },
+      },
+    ])
+
+    const movieHits = response.results[0].hits
+    const gameHits = response.results[1].hits
+
+    expect(movieHits.length).toBe(1)
+    expect(gameHits.length).toBe(14)
+  })
+
+  test('searching on two indexes with scroll pagination', async () => {
+    const customClient = instantMeiliSearch(
+      'http://localhost:7700',
+      'masterKey'
+    )
+
+    const response = await customClient.search<Movies>([
+      {
+        indexName: 'movies',
+        params: {
+          hitsPerPage: 1,
+          page: 1,
+        },
+      },
+      {
+        indexName: 'games',
+        params: {
+          hitsPerPage: 1,
+          page: 1,
+        },
+      },
+    ])
+
+    const movies = response.results[0]
+    const games = response.results[1]
+
+    expect(movies.hits.length).toBe(1)
+    expect(movies.page).toBe(1)
+    expect(movies.nbPages).toBe(3)
+
+    expect(games.hits.length).toBe(1)
+    expect(games.page).toBe(1)
+    expect(games.nbPages).toBe(3)
+  })
+
+  test('searching on two indexes with page selection pagination', async () => {
+    const customClient = instantMeiliSearch(
+      'http://localhost:7700',
+      'masterKey',
+      {
+        finitePagination: true,
+      }
+    )
+
+    const response = await customClient.search<Movies>([
+      {
+        indexName: 'movies',
+        params: {
+          hitsPerPage: 1,
+          page: 1,
+        },
+      },
+      {
+        indexName: 'games',
+        params: {
+          hitsPerPage: 1,
+          page: 1,
+        },
+      },
+    ])
+
+    const movies = response.results[0]
+    const games = response.results[1]
+
+    expect(movies.hits.length).toBe(1)
+    expect(movies.page).toBe(1)
+    expect(movies.nbPages).toBe(6)
+
+    expect(games.hits.length).toBe(1)
+    expect(games.page).toBe(1)
+    expect(games.nbPages).toBe(15)
+  })
+
+  test('searching on two indexes with facet filtering', async () => {
+    const customClient = instantMeiliSearch(
+      'http://localhost:7700',
+      'masterKey'
+    )
+
+    const response = await customClient.search<Movies>([
+      {
+        indexName: 'movies',
+        params: {
+          facetFilters: [['genres:Adventure', 'genres:Action'], ['color:blue']],
+          facets: ['genres', 'color', 'platforms'],
+        },
+      },
+      {
+        indexName: 'movies',
+        params: {
+          page: 0,
+          hitsPerPage: 1,
+          // @ts-ignore
```

Comment:
Out of curiosity, what are you ignoring here?

Reply:
I added the explaination. Instantsearch types telling me its read-only.